README.md

```diff
@@ -1,7 +1,51 @@
 [![Build Status](https://travis-ci.org/EIT-team/PEITS.svg?branch=master)](https://travis-ci.org/EIT-team/PEITS)
 
 # Install guide
-The install guide can be found in the Wiki (correct as of 16.1.2017) [here](https://github.com/EIT-team/PEITS/wiki/Installing-PEITS)
+
+The install guide can be found in the [`INSTALL.md`](./INSTALL.md) file or you
+can use the provided docker container.
+
+Build the container available within the repository using:
+```bash
+docker build -t uclrits/peits:16.04 .
+```
+
+or fetch the latest one available as:
+```bash
+docker fetch uclrits/peits:16.04
+```
+
+Then you can run the container as:
+```bash
+docker run -it --name mypeits -v /full/localpath:/mydata  uclrits/peits:16.04
```

Comment:
As a Docker-illiterate, it's not clear to me what `/full/localpath:/mydata` should be and if I need to replace with something else (David privately told me what `/mydata` is and that the user shouldn't change it).  Just pasting the command the shell seems to work.